cabal.project

```diff
@@ -1,4 +1,5 @@
 packages:
+  filecheck/filecheck.cabal
```

Comment:
A minor thing, but I'd prefer to list `grease`-related libraries first in this `cabal.project` file, since they're the main focus of the repo.